packages/EasyCore/tests/Bridge/Symfony/Valiator/Constraints/AbnValidatorTest.php

```diff
@@ -0,0 +1,296 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Tests\Bridge\Symfony\Valiator\Constraints;
+
+use EonX\EasyCore\Bridge\Symfony\Validator\Constraints\Abn;
+use EonX\EasyCore\Bridge\Symfony\Validator\Constraints\AbnValidator;
+use EonX\EasyCore\Tests\Bridge\Symfony\AbstractSymfonyTestCase;
+use Mockery\MockInterface;
+use stdClass;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Exception\UnexpectedValueException;
+use Symfony\Component\Validator\Violation\ConstraintViolationBuilderInterface;
+
+/**
+ * @covers \EonX\EasyCore\Bridge\Symfony\Validator\Constraints\AbnValidator
+ */
+final class AbnValidatorTest extends AbstractSymfonyTestCase
+{
+    /**
+     * @return string[][]
+     */
+    public function provideAbnWithInvalidCharacters(): array
+    {
+        return [
+            'Contains letter' => ['1234567890a'],
+            'Contains special symbols' => ['1234(678900'],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function provideUnexpectedValues(): array
+    {
+        return [
+            'class without __toString' => [new stdClass(), 'Expected argument of type "string", "stdClass" given'],
+            'integer' => [53004085616, 'Expected argument of type "string", "integer" given'],
+        ];
+    }
+
+    /**
+     * @return string[][]
+     */
+    public function provideValidAbn(): array
+    {
+        return [
+            'Valid Abn #1' => ['53004085616'],
+            'Valid Abn #2' => ['28043145470'],
+            'Valid Abn #3' => ['91724684688'],
+            'Valid Abn #4' => ['10000000000'],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function provideValidEmptyValues(): array
+    {
+        return [
+            'Empty string' => [''],
+            'Null value' => [null],
+        ];
+    }
+
+    /**
+     * @return string[][]
+     */
+    public function provideInvalidAbn(): array
```

Comment:
`provideInvalidAbnValues` to be consistent.

Reply:
Done.